man/systemd.network.xml

```diff
@@ -269,6 +269,26 @@
           if <literal>RequiredForOnline=true</literal>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ActivationMode=</varname></term>
+        <listitem>
+          <para>Sets the activation mode (whether the device should be brought
+          up automatically or not) for the device. Accepts
+          <literal>on</literal>, <literal>off</literal>, or
+          <literal>manual</literal>. Defaults to <literal>on</literal>.</para>
+
+          <para>When set to <literal>on</literal>, networkd manages the device
+          normally and will bring it online as appropriate.</para>
+
+          <para>When set to <literal>manual</literal>, the device will be
+          configured but will remain in the state it was when discovered, and
```

Comment:
I would say "in the UP/DOWN state" here... state is probably a bit too generic and nothing else in the text really clarifies what state is involved here.